src/eravm_error.rs

```diff
@@ -67,4 +67,6 @@ pub enum HeapError {
     StoreOutOfBounds,
     #[error("Trying to read outside of heap bounds")]
     ReadOutOfBounds,
+    #[error("Trying to read/write to heap caused growth which could not be affordable")]
```

Comment:
That message seems odd, maybe something like:
Not enough gas to read/write to heap